### PR TITLE
Shrinks the TokenSelector component

### LINKED ISSUE #39: Shrink the TokenSelector component
It has too much real estate. Shrink it down/move it/whatever

### PR DESCRIPTION
**Pull request by jdmaryville**
Decreased size of token selector panel, increased the size of snippet area.

fixes #39

### CODE REVIEW

File: src/components/AppBody.jsx [hunk 1]
```diff
@@ -140,6 +140,7 @@ class AppBody extends React.Component {
       setTimeout(() => parseReady = true, 1000);
       new Promise((resolve) => resolve(parser(snippet)))
       .then((AST) => {
+        console.log(JSON.stringify(AST));
```
Comment (Hopding) [inline src/components/AppBody.jsx]: Should be able to take this guy out.

File: src/components/LanguageSelector.jsx [hunk 2]
```diff
@@ -4,6 +4,12 @@ import MenuItem from 'material-ui/MenuItem';
 import SelectField from 'material-ui/SelectField';
 
 const LanguageSelector = ({ languages, onChange, selected }) => {
+  const styles = {
```
Comment (Hopding) [inline src/components/LanguageSelector.jsx]: I think we should move this, and the other inline styling, into a css file in the `/styles` folder.

File: src/components/AppBody.jsx (hunk 1 above)
Comment (dane-johnson) [inline src/components/AppBody.jsx]: I think I removed an identical line from a different PR. Make sure you guys are creating your branches off master.

**Comment by dane-johnson** [comment]
http://www.material-ui.com/#/customization/styles

File: src/components/LanguageSelector.jsx (hunk 2 above)
Comment (jdmaryville) [inline src/components/LanguageSelector.jsx]: I agree with moving the in-line styling into the css folder. But on the issue of width between tokens, wont there leave a pretty big amount of empty space between the last token and the length of the panel? Once we get more tokens, I believe then would be a better time to decrease the width between tokens.

File: src/components/AppBody.jsx (hunk 1 above)
Comment (solkaz) [inline src/components/AppBody.jsx]: We need to be more cognizant of taking out our `console.log` statements in our commits.

**Comment by thebho** [comment]
Closing

**Pull request closed without merging**